rfcs/72-editor.md

```diff
@@ -0,0 +1,508 @@
+# Feature Name: `editor-foundation`
+
+## Summary
+
+This RFC outlines the foundation for Bevy's editor. It describes its basic
+components and how they will interact with Bevy's engine. Following this RFC
+will result in a minimum viable product Bevy Editor.
+
+This RFC does NOT cover the particular user interface (UI) design for the
+editor. Instead, it provides the building blocks for a proper UI design, with a
+minimal usable interface to place these components in context.
```

Comment:
This sentence isn't clear to me.

Reply:
How about:

Instead, it provides the minimum set of features required for the Bevy editor, and discusses basic ways these features compose. The full composition of these features, however, is left for a future RFC.

?